threedi_scenario_downloader/downloader.py

```diff
@@ -469,7 +515,8 @@ def transform_to_list(var, length=1):
                     # task is a succes, return download url
                     try:
                         logging.debug(
-                            "Task succeeded, start downloading url: {}".format(
+                            """Task succeeded, start
+                            downloading url: {}""".format(
```

Comment:
Een lange string visueel korter maken door het te veranderen in een multi-line-string is niet gebruikelijk. Het zal hier wel goed gaan, maar mogelijk niet overal. En levert rare logging regels op in dit geval.

Als je `black` over de code heen gooit regelt die het meestal automatisch voor je? Die houdt ook 88 karakter lijnlengte aan i.p.v. de 78 waar je last van had.

Misschien nu gewoon maar even laten staan.

Dit is ook gewoon het probleem dat je krijgt met code die 10 niveau's diep de if/else, for, try/except in gaat, dan heb je weinig ruimte meer over. Oplossing is dan om de boel op het lage niveau wat op te splitsen, maar dat gaat te ver voor deze PR :-)

Binnenkort ga ik als een soort IT-dependance over naar WDS, dus dan kan ik hopelijk makkelijker helpen om allemaal van dit soort projectjes makkelijker programmeerbaar te maken.